src/engine/ruleset/integrations/system/decoders/sysmon-linux.yml

```diff
@@ -269,7 +269,7 @@ normalize:
   - check:
```

Comment:
Could we change this check for this, so that it has the same design as the rest of the checks?

```
 - check:
      - event.code: '16'
```